dspace.php

```diff
@@ -0,0 +1,44 @@
+<?php
+include_once 'autoloader.php';
+
+use BCcampus\OpenTextBooks\Controllers\Catalogue;
+
+include( OTB_DIR . 'assets/templates/partial/style.php' );
+?>
+<style type="text/css">
```

Comment:
I think it had something to do with when the application found itself in a WordPress environment. The CSS  environment was hostile.